### PR TITLE
Add locale help translations

### PR DESCRIPTION
**Pull request by TingALin**
Have added the followings:

- [x] Both the Chinese and English versions of help json files.
- [x] Moved the translated help content into locale files.
- [x] Revised help module types.

Ross:
- [x] Added dynamic imports on language switching & stored language data in localStorage.
- [x] Automatically loads language data from localStorage (bypassing async requests) when present.
- [x] EN as fallback language is always loaded by default.
- [x] Use local `app_version` to check for app updates, wipe local translation data on new version.
- [x] Shrunk translation file size, removed keys in favour of arrays where applicable.

### CODE REVIEW

File: src/contexts/Help/types.ts [hunk 1]
```diff
@@ -4,23 +4,35 @@
 import { ReactNode } from 'react';
 import { MaybeString } from 'types';
 
-export type HelpContentRaw = Array<HelpItemRaw>;
+export type HelpItemLocales = Array<HelpItemLocale>;
```
Comment (rossbulat) [inline src/contexts/Help/types.ts:7]: What was the reasoning for changing these names?

File: src/locale/cn/help.json [hunk 2]
```diff
@@ -0,0 +1,349 @@
+{
+  "help": {
+    "overview": {
```
Comment (rossbulat) [inline src/locale/cn/help.json:3]: I think omit these keys (`overview`, `payouts`, etc) and just have a big list of help items.

**Review by rossbulat — CHANGES_REQUESTED** [review]

File: src/contexts/Help/types.ts (hunk 1 above)
Comment (TingALin) [reply src/contexts/Help/types.ts:7]: Just more intuitive for me

File: src/locale/cn/help.json (hunk 2 above)
Comment (TingALin) [reply src/locale/cn/help.json]: that make sense

**Review by rossbulat — APPROVED** [review]

**Review by rossbulat — DISMISSED** [review]

**Review by rossbulat — APPROVED** [review]

**Pull request merged**